addons/website_sale/static/src/js/website_sale.js

```diff
@@ -473,6 +475,7 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     _onClickAdd: function (ev) {
         ev.preventDefault();
         this.isBuyNow = $(ev.currentTarget).attr('id') === 'buy_now';
+        this.$itemImgContainer = this.$(ev.currentTarget).closest(":has(img:not(.o_product_feature_panel img))");
```

Comment:
strange to have a website_Sale_comparison 'o_product_feature_panel' class here ? what is the bug without it ?

Reply:
Actually, if we get the targeted image from the `closest(":has(img)")` with the "website_sale_comparison" template added in the dom, the image in product comparison panel will be used instead **(issue reported in task)**.